src/components/Garden/ModalFlows/ChallengeProposalScreens/ChallengeRequirements.js

```diff
@@ -20,6 +21,9 @@ import { getDisputableAppByName } from '@utils/app-utils'
 import iconError from '@assets/iconError.svg'
 import iconCheck from '@assets/iconCheck.svg'
 
+import { useHistory } from 'react-router-dom'
```

Comment:
As a similar comment, we generally set external library imports first (can place it after React import)